packages/core/core/src/services/document-service/middlewares/defaults/locales.ts

```diff
@@ -41,7 +41,14 @@ export const localeToLookup: Middleware = async (ctx, next) => {
  * Translate locale status parameter into the data that will be saved
  */
 export const localeToData: Middleware = async (ctx, next) => {
-  if (!isLocalizedContentType(ctx.uid)) return next(ctx);
+  if (!isLocalizedContentType(ctx.uid)) {
```

Comment:
Instead of a custom scenario here, could we add this to the sanitization and validation? That way if preferred (like in most cases) it can throw an error rather than silently removing it

Reply:
Actually, thinking about this specific part more, we should just throw an error because it's not a valid field. And the sanitization and validation would be a bonus, and prevent it from ever being here in the first place.

Reply:
that makes sense, will do that.
I believe this might involve typing work too then!, as locale is a field in the schema, that should not be included in Data